crowbar_framework/lib/crowbar/checks/network.rb

```diff
@@ -67,19 +67,11 @@ def ip_configured?
       end
 
       def firewall_disabled?
-        if system("sudo LANG=C iptables -n -L | grep -qvE '^$|^Chain [^ ]|^target     prot'")
-          return false
-        end
-
-        true
+        return ! system("sudo LANG=C iptables -n -L | grep -qvE '^$|^Chain [^ ]|^target     prot'")
```

Comment:
Maybe these spaces between "target" and "prot" and be expressed also as a regex (I know that is not related with this PR)